tests/beacon/conftest.py

```diff
@@ -349,3 +363,25 @@ def max_attestation_count():
 @pytest.fixture
 def initial_fork_version():
     return SERENITY_CONFIG.INITIAL_FORK_VERSION
+
+
+#
+# genesis
+#
+@pytest.fixture
+@to_tuple
+def genesis_validators(init_validator_keys,
+                       init_randao,
+                       deposit_size):
+    return [
```

Comment:
Same here, the `to_tuple` decorator should either be dropped in favor of just using a tuple comprehension **or** you should convert the inner body to use a `yield` pattern if you want to keep the `to_tuple` decorator.